projects/pgai/pgai/vectorizer/loading.py

```diff
@@ -15,13 +16,21 @@ class LoadedDocument:
 
 
 def guess_filetype(file_like: BytesIO, file_path: str | None = None) -> str | None:
-    guess = filetype.guess(file_like)  # type: ignore
+    guess = filetype.guess(file_like)  # type: ignore[reportUnknownArgumentType,reportUnknownMemberType]
```

Comment:
This error wasn't popping previously but now it does. Im just skipping. Other solution would be to do explicit casts.